en/flight_modes_fw/altitude.md

```diff
@@ -5,63 +5,57 @@
+
+The yaw stick can be used to increase/reduce the yaw rate of the vehicle in turns.
+If left at center the controller does the turn coordination by itself, meaning that it will apply the necessary yaw rate for the current roll angle to perform a smooth turn.
 
 The diagram below shows the mode behaviour visually (for a [mode 2 transmitter](../getting_started/rc_transmitter_receiver.md#transmitter_modes)).
 
 ![Altitude Control FW](../../assets/flight_modes/altitude_fw.png)
 
 ## Technical Summary
 
-Manual mode like [Stabilized mode](../flight_modes_fw/stabilized.md) but with altitude stabilization (centered sticks put vehicle into straight and level flight and maintain current altitude).
+Altitude mode like [Stabilized mode](../flight_modes_fw/stabilized.md) but with altitude stabilization.
+Airspeed is also stabilized if an airspeed sensor is present.
 The vehicle course is not maintained, and can drift due to wind.
 
 - Centered Roll/Pitch/Yaw inputs (inside deadband):
-  - Autopilot levels vehicle/wings and maintains altitude.
-  - Throttle stick controls the airspeed of the aircraft if an airspeed sensor is connected. Without an airspeed sensor the user cannot control throttle (in which case the vehicle will fly level at trim throttle ([FW_THR_TRIM](../advanced_config/parameter_reference.md#FW_THR_TRIM)), increasing or decreasing throttle as needed to climb or descend).
+  - Autopilot levels vehicle and maintains altitude and airspeed.
 - Outside center:
   - Pitch stick controls altitude.
-  - Throttle stick controls the airspeed of the aircraft (as for centered Roll/Pitch/Yaw inputs).
+  - Throttle stick controls the airspeed of the aircraft if an airspeed sensor is connected. Without an airspeed sensor the vehicle will fly level at trim throttle ([FW_THR_TRIM](../advanced_config/parameter_reference.md#FW_THR_TRIM)), increasing or decreasing throttle as needed to climb or descend.
   - Roll stick controls roll angle. Autopilot will maintain [coordinated flight](https://en.wikipedia.org/wiki/Coordinated_flight).
-    This is same as in [Stabilized mode](../flight_modes_fw/stabilized.md).
-  - Yaw stick actuates the rudder (signal will be added to the one calculated by the autopilot to maintain [coordinated flight](https://en.wikipedia.org/wiki/Coordinated_flight)).
-    This is same as in [Stabilized mode](../flight_modes_fw/stabilized.md).
+  - Yaw stick adds an additional yaw rate setpoint (added to the one calculated by the autopilot to maintain coordinated flight).
+  Can be used to manually change the side slip of the vehicle.
 - Manual control input is required (such as RC control, joystick).
-  - Roll: Assistance from autopilot to hold altitude against wind.
-  - Pitch: Assistance from autopilot to stabilize the attitude.
-    Position of RC stick maps to the orientation of vehicle.
-  - Yaw: Input is sent directly to control allocation.
-  - Throttle: Assistance from autopilot to hold altitude against wind.
```

Comment:
FYI, this stuff came from the overview table which has S+, M, S, Srate and so on. I deleted that and copied it here. I'm cool for it to be gone if incorrect or not needed.